packages/@okta/vuepress-site/guides/sign-into-web-app/define-callback/springboot/define-route.md

```diff
@@ -1,3 +1,7 @@
-The Okta Spring Boot SDK configures and hosts this route for you. By default, the route is hosted at `/authorization-code/callback`.
+The Okta Spring Boot Starter configures and hosts this route for you. By default, this route is hosted at `/login/oauth2/code/okta`.
```

Comment:
Ah, that's the default if you install Okta's Spring Boot Starter and *don't* provide any additional config?

Reply:
Correct, this is the default spring route, if you configured Okta along with another provider you could have `/login/oauth2/code/<other-provider>`  All of our samples just make the assumption there is a single provider